pytest_splunk_addon/splunk.py

```diff
@@ -322,3 +273,60 @@ def splunk_web_uri(splunk):
     uri = f'http://{splunk["host"]}:{splunk["port_web"]}/'
```

Comment:
it can be http or https? with cloud it's https I believe. you'll have to check if ssl is enabled in web.conf to see if you should use http or https, then you have to check for the "root endpoint" as well since it's not always '/'.

Reply:
splunk_web_uri is not in the scope of this version of pytest-splunk-addon as we're only testing the field extractions & CIM mapping through the management port